poi-ooxml/src/test/java/org/apache/poi/openxml4j/opc/TestZipPackage.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.poi.openxml4j.opc;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class TestZipPackage {
+
+    @Test
+    void testDefaultDirectory(@TempDir Path tempDir) {
```

Comment:
Unfortunately there isn't much to test here, as the default behavior is to clean up after the temp file immediately after it is written. You can set a breakpoint in `OpcPackage.closeImpl` and inspect for yourself, but capturing that in a unit test would likely mean trying to introduce a lot of testing overhead that, quite frankly, isn't needed.

Maybe desired, but it's not really worth the effort for something POI might not want to accept as a testing utility contribution. Feel free to jump in if you disagree, @MariusVolkhart

Reply:
I should add that I'll be modifying the IOPS test to capture this and make assertions against writes to both this temp file and the Unpacker temp file we create.